plenum/common/constants.py

```diff
@@ -120,7 +119,7 @@
 NYM = PlenumTransactions.NYM.value
 GET_TXN = PlenumTransactions.GET_TXN.value
 
-openTxns = (GET_TXN)
+openTxns = (GET_TXN, )
```

Comment:
Looks like we don't need this constant at all anymore

Reply:
removed